<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoPusherClass</class>
 <widget class="QWidget" name="DemoPusherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DemoPusher</string>
  </property>
  <widget class="QFrame" name="m_videoShow">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="LiveButton_">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始直播</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PreviewButton_">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭预览</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="streamId_">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="postUrl">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>150</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>streamid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>150</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>postUrl</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DemoPusher.qrc"/>
 </resources>
 <connections/>
</ui>
